app/views/shared/registrations/_action_links_panel.html.erb

```diff
@@ -30,7 +30,8 @@
       </li>
     <% end %>
 
-    <% if display_view_confirmation_letter_link_for?(resource) %>
+    <!-- TODO: put back the check when internal route exists https://eaflood.atlassian.net/browse/RUBY-786 -->
+    <% if false #display_view_confirmation_letter_link_for?(resource) %>
```

Comment:
Why not modify `display_view_confirmation_letter_link_for?(resource)` instead? It would be more consistent with the others.

Reply:
yep!! :) Thanks for catching this